app/views/comments/_comment.html.slim

```diff
@@ -0,0 +1,18 @@
+li.row.comment
+  .col-sm-3.col-md-2.text-center-xs
+    p
+      img.user_img src='#{comment.user.image_url}'
```

Comment:
do you know about image_tag? read about it and tell about it to other developers